mobilizon_bots/publishers/telegram.py

```diff
@@ -1,50 +1,111 @@
 import requests
 
+from mobilizon_bots.config.config import settings
+
 from .abstract import AbstractPublisher
+from .exceptions import (
+    InvalidBot,
+    InvalidCredentials,
+    InvalidEvent,
+    InvalidResponse,
+    InvalidSettings,
+)
 
 
 class TelegramPublisher(AbstractPublisher):
-    def post(self) -> bool:
-        chat_id = self.credentials["chat_id"]
-        text = self.event["text"]
-        token = self.credentials["token"]
-        post_params_kwargs = self.event.get("post_params_kwargs")
+    def post(self):
+        attrs = self.get_attrs_from_conf()
         res = requests.post(
-            url=f"https://api.telegram.org/bot{token}/sendMessage",
-            params=dict(chat_id=chat_id, text=text, **post_params_kwargs),
+            url=f"https://api.telegram.org/bot{attrs['token']}/sendMessage",
+            params={"chat_id": attrs["chat_id"], "text": self.message},
         )
-        return self._validate_response(res)
+        try:
```

Comment:
qui sei tornato a fare il return di booleani che avevamo rimosso. Io questo try catch lo metterei esterno e condiviso tra tutti i publisher. O in alternativa ritorni un Optional della response che se è None vuol dire che ha fallito.

Reply:
@chobeat quindi dici meglio usare una `try/catch` qui al posto dell'`if/else`?
https://github.com/Tech-Workers-Coalition-Italia/mobilizon-bots/blob/master/mobilizon_bots/publishers/__init__.py#L23-L26

Reply:
sì, con le eccezioni sollevate che siano più o meno standardizzate tra i vari publisher/notifier

Reply:
Claro, alla fin fine basterà usare le eccezioni definite in `publishers/exceptions.py`. Allora rimedio asap anche qui.